vault/dendron.topic.links.md

```diff
@@ -1,97 +1,80 @@
 ---
 id: 3472226a-ff3c-432d-bf5d-10926f39f6c2
 title: Links
-desc: 'Dendron supports many ways of linking notes and files'
-updated: 1658768400279
+desc: Create links to various locations using basic and extended syntax
+updated: 1659082798415
 created: 1595003088839
+config:
+  global:
+    enableChildLinks: false
 ---
 
 ## Summary
 
 {{fm.desc}}.
 
-## Types
-- Markdown links (can link to any file):
-    - `[link text](dendron.note.name.md)` -> [link text](dendron.note.name.md)
-    - `[link text](./path/to/filename.md)` -> [link text](./path/to/filename.md)
-    - `[link text](assets/filename.pdf)` -> [link text](assets/filename.pdf)
-- Wiki links: `[[dendron.note.name]]` -> [[dendron.note.name]]
-- Image links: `![image alt text](http://example.com/or/path/to/image)`
-
-## Concepts
-
-### Note Files
-
-A note file is a Dendron Markdown note.
-
-### Regular Files
-
-Regular files are any assets that are not Dendron Markdown notes. 
+Dendron provides many ways to create links between notes and assets. This range from regular markdown links to extended syntax such as wikilinks, file links, and much more.
 
 ## Features
```

Comment:
so for features, i'm particular about the wording. this is what I would have for this section. ideally, we'd have a GIF/loom video demonstrating all the features as well

--- 

## Features

### Link to anything

Whether the resource you're linking to is on your hard drive or on the internet, Dendron can help you link to it.

<!-- #TODO(wiki link, file link, url link) -->

### Links in Context

See at a glance all notes that link to the current note with backlinks. 

<!-- #TODO(backlink panel) -->

### Link Previews

Hover over links to get rich previews to get the context before opening a link.

<!-- #TODO(hover over link, hover over backlink) -->

### Quick Link Creation

Use autocomplete and `Insert Note Link` to create a link to an existing note. Use commands like `Copy Note Link` and `Copy File Link` to create a link from an open note or file. 

<!-- #TODO(insert note link, copy note link) -->

### Quick Link Traversal

Navigate links without leaving the keyboard. If the link is a website or a custom file format, Dendron will use your system default program to open the link.

<!-- #TODO(open a note, open an external link) -->

### Visualize links

Connect the dots using the [[Graph View|dendron://dendron.dendron-site/dendron.topic.graph-view]]

<!-- TODO: graph view -->